BUILD

```diff
@@ -61,31 +61,16 @@ cc_library(
         "src/svr_training",
     ],
     linkopts = select({
-        "@bazel_tools//src/conditions:windows": [
-            # Windows Link Opts
-            "-LIBPATH:C:\\\\boost\\\\stage\\\\lib",
-        ],
-        "@bazel_tools//src/conditions:darwin_x86_64": [
-            # Mac Link Opts
-        ],
         "//conditions:default": [
             # Linux Link Opts
-            "-lboost_system",
-            "-lboost_filesystem",
         ],
     }),
     visibility = ["//visibility:public"],
     deps = select({
-        "@bazel_tools//src/conditions:windows": [
-            # Windows Dependencies
-            "@pffft_lib_win//:pffft_win",
```

Comment:
This dep should remain.

Reply:
ah, yes, I accidentally killed pffft as well. Should be resolved now